tests/test_attention.py

```diff
@@ -51,6 +51,22 @@ def test_CrossAttention():
     assert out.shape == test_tensor1.shape
     del attention
 
+def test_MultiScaleAttention():
+
+    test_tensor1 = torch.randn(96,8,56,56)
+    test_tensor2 = torch.randn(768,8,14,14)
+    thw  = [2,2,2]
```

Comment:
I did one more review round; 
i guess you will need to create 5 dimensional input tensors (Batch,Channels,Time,Height,width) according to the authors code